Automattic/jetpack
# Atomic: Supress Development Version status for weekly releases

- Always consider Atomic versions as not development
- Add logic to not suppress on versions provided by Jetpack Beta




As we move to weekly releases to the Atomic platform, we are using a 10.3-a.1, etc style version number to denote that it is a weekly release (while keeping our w.org-provided version scheme the same).

We don't want to display any notices that we provide for having a version of the plugin that is in development. This PR adds tests and logic to suppress that this is a development version for the weekly release on the Atomic platform.

#### Changes proposed in this Pull Request:

* Suppresses "Development Version" notice on the Atomic platform, except when it is a version provided by the Beta plugin (e.g. folder of `jetpack-dev/`

#### Jetpack product discussion


p1634668120195300-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On a WoA site, install the beta plugin and install this version. You should still see the development notice since it was provided by the beta version.
* Rename the folder from `jetpack-dev` to anything else, like `jetpack-testing`. The development version notice should not appear.
